GNUmakefile

```diff
@@ -5,6 +5,7 @@ VERSION=1.0.0
 BINARY=terraform-provider-${NAME}
 OS_ARCH=darwin_amd64
 # Switch between the two OS architecture according to own computer's architecture when working locally (if needed)
+# arm64 is the OS_ARCH for m1 or m2 apple processor
```

Comment:
Put this, let's resolve it dynamically 
`
UNAME_P := $(shell uname -p)
ifeq ($(filter %86,$(UNAME_P)),)
	OS_ARCH=darwin_amd64
endif
ifneq ($(filter arm%,$(UNAME_P)),)
	OS_ARCH=darwin_arm64
endif
$(info Current architecture (OS_ARCH) is $(OS_ARCH))
`

Reply:
UNAME_P := $(shell uname -p)
ifeq ($(filter %86,$(UNAME_P)),)
	OS_ARCH=darwin_amd64
endif
ifneq ($(filter arm%,$(UNAME_P)),)
	OS_ARCH=darwin_arm64
endif
$(info Current architecture (OS_ARCH) is $(OS_ARCH))